develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,205 @@
+
+   ```json
+   {
+     "Parameters": {
+       "TIDB_HOST": "{tidb_server_host}",
+       "TIDB_PORT": "4000",
+       "TIDB_USER": "root",
+       "TIDB_PASSWORD": "{password}"
+     }
+   }
+   ```
+
+   Replace the placeholders in `{}` with the values obtained in the **Connect** window.
+
+2. [Configure Step.1 environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) in Lambda Function..
+
+</div>
+
+</SimpleTab>
+
+### Step 4: Run the sample application locally
+
+1. Install the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html).
+
+2. Run the following command to start the application:
+
+    ```bash
+    sam local invoke --env-vars env.json -e event.json "tidbHelloWorldFunction"
+    ```
+
+3. Check the output in the terminal. If the output is similar to the following, the connection is successful:
+
+    ```bash
+    {"statusCode":200,"body":"{\"results\":[{\"Hello World\":\"Hello World\"}]}"}
+    ```
+
+## Key Code Snippets
+
+Complete code snippets are available in the [tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
+
+You can refer to the following key code snippets to complete your application development.
+
+### Connect to TiDB
+
+```typescript
+import mysql from 'mysql2';
+
+const pool = mysql.createPool({
+  host,
+  port,
+  user,
+  password,
+  database,
+  ssl: {
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true,
+  },
+  waitForConnections: true,
+  connectionLimit: 1,
+  maxIdle: 1, // max idle connections, the default value is the same as `connectionLimit`
+  idleTimeout: 60000, // idle connections timeout, in milliseconds, the default value 60000
+  queueLimit: 0,
+  enableKeepAlive: true,
+  keepAliveInitialDelay: 0,
+});
+```
+
+### Insert data
+
+```typescript
+const player = ['1', 10, 500];
+await pool.execute(
+  `INSERT INTO player (id, goods, coins) VALUES (${player[0]}, ${player[1]}, ${player[2]})`
+);
+```
+
+Refer to [Insert data](/develop/dev-guide-insert-data.md) for more information.
+
+### Query data
+
+```typescript
+const results = await pool.execute('SELECT count(*) FROM player');
+console.log(results);
+```
+
+Refer to [Query data](/develop/dev-guide-get-data-from-single-table.md) for more information.
+
+### Update data
+
+```typescript
+const player = ['1', 10, 500];
+await pool.execute(
+  `UPDATE player SET goods = goods + ${play[0]}, coins = coins + ${play[1]} WHERE id = ${play[2]}`
+);
+```
+
+Refer to [Update data](/develop/dev-guide-update-data.md) for more information.
+
+### Delete data
+
+```typescript
+await pool.execute('DELETE FROM player WHERE id = 1');
+```
+
+Refer to [Delete data](/develop/dev-guide-delete-data.md) for more information.
+
+## Considerations
+
+- Complete code snippets and how to run them, see [tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
```

Comment:
Remove it or add it to the top of this document like:

<img width="1056" alt="image" src="https://github.com/pingcap/docs/assets/5086433/d7e7a190-961e-48f3-ae3c-fdb0d952ae27">